home/ivp-planning/user-interface/hierarchy-panel.md

```diff
@@ -0,0 +1,65 @@
+
+## The elements of the panel:
+
+|                                                           |                                                                                                                                                   |
+| --------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ![](../../../.gitbook/assets/Hierarchy_search.jpg)        | The search bar makes it possible to only display such machines or objects which names contain the entered text (the search is not case sensitive) |
+| ![](../../../.gitbook/assets/Hierarchy_folder_button.jpg) | Adds a new folder to the hierarchy                                                                                                                |
+| ![](../../../.gitbook/assets/Hierarchy_color.jpg)         | Visualisation of the [color](../machines/highlighting-objects.md) set for the object(s)                                                           |
+| ![](../../../.gitbook/assets/Hierarchy_lock.jpg)          | Locks or unlocks the currently selected object(s)                                                                                                 |
+| ![](../../../.gitbook/assets/Hierarchy_visibility.jpg)    | Toggles the visibility of the currently selected object(s)                                                                                        |
+
+## Folders
+
+Folders can be used to structure projects. They can only be accessed in the hierarchy panel, however locking or hiding them will affect all objects inside - also in the [2D- ](../user-interface/the-2d-panel.md)and [3D-panel](../user-interface/the-3d-panel.md).
+
+### Creating folders:
+
+To create a folder, **press the "new folder" button** in the top of the hierarchy panel and a new folder will be added.
+
+![](../../../.gitbook/assets/Hierarchy_new_folder.jpg)
+
+{% hint style="info" %}
+The position of the new folder in will be in the topmost layer of the hierarchy. It can then be sorted into/under other folders or objects. 
+{% endhint %}
+
+### Moving objects in the hierarchy panel:
+
+To move objects or other object in the hierarchy, [select](../machines/selecting-and-moving-objects.md#selecting-objects-via-the-hierarchy-panel) their entries in the hierarchy panel and drag them on the folder/object name. A white frame will indicate where the entries will be moved to and they will be sorted into other folders or objects if applicable.
+
+![](../../../.gitbook/assets/Hierarchy_move.jpg)
+
+## Lock/unlock objects
+
+To prevent most [object interactions](../machines/README.md) for one or more objects, [select](../machines/selecting-and-moving-objects.md#selecting-objects-via-the-machine-list-panel) them and press the <img src="../../../.gitbook/assets/Hierarchy_lock.jpg" alt="" data-size="line"> **Lock/Unlock Machine** button.
```

Comment:
We'll need to refactor this and the hint below. "most object interactions" is very vague and the hint mentioning that a locked object can still be moved is flat out wrong. A locked object can not be selected from the 2D & 3D panel and can not be moved. That's all locking does.

Reply:
the hint missed the bit that those actions are available in the hierarchy.

changed the wording to be correct.